pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -260,7 +260,9 @@ spec:
                 command:
                 - /bin/sh
                 - -c
-                - zfs set io.openebs:livenesstimestamp="$(date +%s)" cstor-$OPENEBS_IO_CSTOR_ID
+                ## timeout 30 is added to exit the command forcefully with non-zero exit code if command takes
```

Comment:
Make this 300s to avoid false positives.

Reply:
Updated settings.